tests/test_client_request.py

```diff
@@ -88,6 +88,11 @@ def test_method3(make_request: Any) -> None:
     assert req.method == "HEAD"
 
 
+def test_method3_invalid(make_request: Any) -> None:
+    with pytest.raises(ValueError):
```

Comment:
The best practice is to always use the `match=` argument with `pytest.raises()`.